### PR TITLE
[MIG] 14->16 l10n_ir

### PR DESCRIPTION
**Pull request by saeed-raeisi**
Tested in Odoo 16 Enterprise

- l10n_ir_accounting(The tax report has been removed and will be added soon)
- l10n_ir_base(This module has been removed and will be added again after further review)
- l10n_ir_hr_contract(The setting of default values ​​has been added and Iran's daily salary can be applied)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git
+    ${HOME}/maintainer-quality-tools
+  - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - travis_install_nightly
+
+script:
+  - travis_run_tests
+
+after_success:
+  - travis_after_tests_success
```
Comment (sbidoul) [inline .travis.yml]: We don't use travis anymore. Removing this file should help.

File: .pylintrc-mandatory [hunk 2]
```diff
@@ -26,7 +26,6 @@ enable=anomalous-backslash-in-string,
     duplicate-id-csv,
     duplicate-key,
     duplicate-xml-fields,
-    duplicate-xml-record-id,
```
Comment (sbidoul) [inline .pylintrc-mandatory:29]: I suggest keeping this check.

**Comment by sbidoul** [comment]
Hi @saeed-raeisi. I changed the default branch and made a couple of comments on your PR.

File: .pylintrc-mandatory (hunk 2 above)
Comment (saeed-raeisi) [reply .pylintrc-mandatory:29]: > I suggest keeping this check.

The problem I have is related to setting the default accounts: `account.chart.template`

Before I have to create the accounts in
`account.account.template.csv`

But here is the field
`chart_template_id/id`
I have to fill

duplicate-xml-record-id --> `id=l10n_ir_chart_template` in:
l10n_ir_chart_post_data.xml
l10n_ir_chart_data.xml

**Comment by saeed-raeisi** [comment]
> Hi @saeed-raeisi. I changed the default branch and made a couple of comments on your PR.

Thankful

**Pull request merged**